gammapy/data/observations.py

```diff
@@ -79,18 +79,17 @@ def __init__(
         self.obs_filter = obs_filter or ObservationFilter()
 
     @property
-    def available_irfs(self):
-        """Which irfs are available"""
-        available_irf = []
-
-        for irf in ["aeff", "edisp", "psf", "bkg"]:
-            available = self.__dict__.get(irf, False)
-            available_hdu = self.__dict__.get(f"_{irf}_hdu", False)
-
+    def available_hdus(self):
```

Comment:
Why not keep both `available_irfs` and `available_hdus`? I think there might be use-case for both...